ModularTegustation/fishing/code/fishing_items/fishing_refinery.dm

```diff
@@ -0,0 +1,56 @@
+/obj/structure/fish_refinery
+	name = "Fish Refinery"
+	desc = "A machine used to refine any fish into raw PE."
+	icon = 'ModularTegustation/fishing/icons/fishmachines.dmi'
+	icon_state = "fish_refinery"
+	anchored = TRUE
+	density = TRUE
+	resistance_flags = INDESTRUCTIBLE
+	var/refine_timer = 2 MINUTES
+	var/timeleft
+	var/loaded = FALSE
+	var/fish_amount = 0
+
+/obj/structure/fish_refinery/Initialize()
+	. = ..()
+	GLOB.lobotomy_devices += src
+
+/obj/structure/fish_refinery/Destroy()
+	GLOB.lobotomy_devices -= src
+	if(loaded)
+		new /obj/item/food/fish(get_turf(src))
+	return ..()
+
+/obj/structure/fish_refinery/examine(mob/user)
+	. = ..()
+	if(loaded==FALSE)
```

Comment:
placing an exclamation point before a variable name in an if() check is a "better" way to handle if(X == FALSE) checks
```suggestion
	if(!loaded)
```